ocs_ci/ocs/utils.py

```diff
@@ -1266,3 +1267,50 @@ def get_primary_cluster_config():
     for cluster in ocsci_config.clusters:
         if cluster.MULTICLUSTER["primary_cluster"]:
             return cluster
+
+
+def get_provider_cluster_config():
+    """
+    Get the provider cluster config object in a MS scenario
+
+    Returns:
+        framework.config: provider cluster config object from config.clusters
+    """
+    for cluster in ocsci_config.clusters:
+        if cluster.ENV_DATA.get("cluster_type") == "provider":
+            return cluster
+
+
+def run_on_cluster_type(cluster_type):
```

Comment:
If we want to use more than one consumer, we need to add another param like `consumer_num` or `consumer_name`. But I am not sure if it's mandatory for now.